homework/Truong_Duong/lesson_4/53_Maximum_subarray.js

```diff
@@ -0,0 +1,20 @@
+// Solution for https://leetcode.com/problems/maximum-subarray/
+
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+ var maxSubArray = function(nums) {
+    const size = nums.length;
+    const prefixSum = new Array(size).fill(0);
+    
+    let ans = Number.MIN_SAFE_INTEGER, min = 0;
+          
+    for (let i = 0; i < size; i++) {
+        prefixSum[i] = i == 0 ? nums[i] : prefixSum[i-1] + nums[i];
```

Comment:
recommend: in a coding interview, I would not recommend using ternary operation, it is hard to read/ debug